nemo_rl/algorithms/distillation.py

```
@@ -0,0 +1,888 @@
+        else:
+            if inference_gpus_per_node is None:
+                inference_gpus_per_node = cluster_config["gpus_per_node"]
+            if inference_nodes is None:
+                inference_nodes = cluster_config["num_nodes"] // 2
+
+        # validate resources
+        if inference_gpus_per_node > cluster_config["gpus_per_node"]:
+            raise ValueError(
+                f"Inference GPUs per node ({inference_gpus_per_node}) cannot be greater than "
+                f"total GPUs per node ({cluster_config['gpus_per_node']})"
+            )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)